game-engine/assets/example-state/state.json

```diff
@@ -2,6 +2,7 @@
   "currentRound": 0,
   "maxRounds": 200,
   "pushbackDamage": 4,
+  "opponentsLastCommand": "nothing",
```

Comment:
I don't see any changes to the text state file? Unless I am missing something, please have a look at adding the logic for that in the Text Renderer. The text state is generally used by players using c++ who do not want to parse json, and should contain all the information that the json state contains. 

If the text renderer is giving you any trouble, @blaarkies or myself will be happy to assist :)

Reply:
Thanks.  Will do!
I'll try to infer how the text representation works first and get back to you if I have any problems updating it.